hipcheck/src/cli.rs

```diff
@@ -389,6 +389,9 @@ pub enum Commands {
 	/// Print the JSON schema for output of a specific `check` command.
 	Schema(SchemaArgs),
 	/// Initialize Hipcheck config file and script file locations.
+	///
+	/// NOTE: The `--data` and `--config` flags specify where to install the
+	/// extra files Hipcheck needs.
```

Comment:
Should this comment explain that the `data` and `config` paths can also be resolved by env var or platform default?